server/util/util.go

```diff
@@ -0,0 +1,99 @@
+package util
+
+import (
+	"bytes"
+	"encoding/json"
+	"github.com/pkg/errors"
+	"github.com/stretchr/testify/assert"
+	"io"
+	"net/http"
+	"net/http/httptest"
+)
+
+const (
+	// ContentTypeJSON string associated with content-type json
+	ContentTypeJSON ContentType = "application/json"
+
+	// ContentTypeHTML string associated with content-type html
+	ContentTypeHTML ContentType = "text/html"
+
+	// ContentTypePlain
+	ContentTypePlain ContentType = "text/plain; charset=utf-8"
+)
+
+// ContentType type for http Content-Type
+type ContentType string
```

Comment:
Nit: Does the `type` need go be exported?

Reply:
I'm using it [here](https://github.com/bakurits/mattermost-plugin-github/blob/b4f906611637384984d941bdfcf6ca81b0537755/server/api_test.go#L43)

Reply:
Surely `ContentTypeJSON` need to be exported. But does `ContentType`?

Reply:
Ah sorry for the misunderstanding. I'll change that